providers/flagd/README.md

```diff
@@ -44,16 +44,16 @@ func main() {
 ### Using flagd.FromEnv()  
 By default the flagd provider will read non-empty environment variables to set its own configuration with the lowest priority. Use the `flagd.FromEnv()` option as an argument for the `flagd.NewProvider()` method to give environment variables a higher priority.
 
-| Option name           | Environment variable name      | Type      | Options      | Default   |
-|-----------------------|--------------------------------|-----------|--------------|-----------|
-| host                  | FLAGD_HOST                     | string    |              | localhost |
-| port                  | FLAGD_PORT                     | number    |              | 8013      |
-| tls                   | FLAGD_TLS                      | boolean   |              | false     |
-| socketPath            | FLAGD_SOCKET_PATH              | string    |              |           |
-| certPath              | FLAGD_SERVER_CERT_PATH         | string    |              |           |
-| cache                 | FLAGD_CACHE                    | string    | lru,disabled | lru       |
-| maxCacheSize          | FLAGD_MAX_CACHE_SIZE           | int       |              | 1000      |
-| maxEventStreamRetries | FLAGD_MAX_EVENT_STREAM_RETRIES | int       |              | 5         |
+| Option name           | Environment variable name      | Type    | Options            | Default   |
+|-----------------------|--------------------------------|---------|--------------------|-----------|
+| host                  | FLAGD_HOST                     | string  |                    | localhost |
+| port                  | FLAGD_PORT                     | number  |                    | 8013      |
+| tls                   | FLAGD_TLS                      | boolean |                    | false     |
+| socketPath            | FLAGD_SOCKET_PATH              | string  |                    |           |
+| certPath              | FLAGD_SERVER_CERT_PATH         | string  |                    |           |
+| cache                 | FLAGD_CACHE                    | string  | lru, mem, disabled | lru       |
```

Comment:
Can you explain why you added this new cache type?

I guess it's not a problem, but I wonder why anyone would use it over LRU.

Reply:
We always supported in-memory cache but lacked documentation under environment configuration section - https://github.com/open-feature/go-sdk-contrib/blob/6545b4c16db57114456c58a3eb2966ec461ae9f9/providers/flagd/README.md#using-flagdfromenv